tests/test_byte_order.rs

```diff
@@ -24,13 +24,23 @@ macro_rules! test_primitive {
 
             assert_eq!(&bytes, &$serialized);
         }
-    }
+    };
 }
 
 // These tests check whether byte ordering isn't changed due to mistake in implementation
 test_primitive!(u16, U16, 42, [42, 0, 0, 42]);
 test_primitive!(u32, U32, 42, [42, 0, 0, 0, 0, 0, 0, 42]);
-test_primitive!(u64, U64, 42, [42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 42]);
+test_primitive!(
+    u64,
+    U64,
+    42,
+    [42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 42]
+);
 test_primitive!(i16, I16, 42, [42, 0, 0, 42]);
 test_primitive!(i32, I32, 42, [42, 0, 0, 0, 0, 0, 0, 42]);
-test_primitive!(i64, I64, 42, [42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 42]);
+test_primitive!(
+    i64,
+    I64,
+    42,
+    [42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 42]
+);
```

Comment:
Hmm, this is why I'm not a huge fan of auto formatting. This one looked better before. Could you revert this specific change, please?

Reply:
The auto-format can be easy configurable and in most cases it work quite well. It's done automatically on every save in my editor. So not pleasant  to work with not formatted code. I think the last commit should make both of us happy.